wiki/Beatmap/en.md

```diff
@@ -1,38 +1,52 @@
-# Beatmaps
+---
+tags:
+  - mapset
+  - beatmapset
+---
 
-*Not to be confused with [Difficulties](/wiki/Difficulties).*
-
-**Beatmaps** are game levels created for songs and stored in text files with [the `.osu` extension](/wiki/osu!_File_Formats/Osu_(file_format)). They have different structure and sometimes may be played only in one [game mode](/wiki/Game_mode).
+# Beatmap
 
 ## Overview
 
-At the core, a beatmap consists of [hit objects](/wiki/Hit_object) of a certain [game mode](/wiki/Game_mode), which are set up accordingly to the [timing](/wiki/Glossary#timing-section) tailored to a specific song. Beatmaps have different aspects, which directly affect user experience.
+A **beatmap** (sometimes called *beatmapset*) is a set of game levels, or [difficulties](#difficulty), that are composed of various [hit objects](/wiki/Hit_object) and represent a single song. It also includes other components, all packed in an archive with [the `.osz` extension](/wiki/osu!_File_Formats/Osz_(file_format)):
+
+- The song itself, stored in MP3 or OGG format.
+- [Background images](/wiki/Beatmap/Background), or a video, acting as a playfield.
+- (optional) [Custom hitsounds](/wiki/Beatmapping/Hitsound) for arrangement and improved aural feedback.
+- (optional) [Storyboard](/wiki/Storyboards) with motion graphics and special effects, or a background story or theme for the song.
+- (optional) [Custom skin](/wiki/Skinning), which changes the appearance of interface and gameplay elements.
+
+Some of the visual and aural features of a beatmap may be disabled through the [visual settings](/wiki/Visual_Settings) overlay.
+
+## Difficulty
 
-The hit objects of a beatmap compose [patterns](Pattern), which also vary per game mode and set the tone for overall difficulty, and how a beatmap should be played. The process of playing the beatmap is altered by various [difficulty settings](/wiki/Beatmap_Editor/Song_Setup#difficulty) set by the beatmap's author and [game modifiers](/wiki/Game_modifier) toggled by a player.
+*Main article: [Difficulties](/wiki/Difficulties)*
 
-From the auditory perspective, a beatmap can augment the song by providing one or multiple sets of audio samples, or [hitsounds](/wiki/Beatmapping/Hitsound), that serve as an additional layer of feedback.
+A difficulty is a file which describes the placement of hit objects, hitsounds, and special effects like [kiai time](/wiki/Kiai_time). It also contains [difficulty settings](/wiki/Beatmap_Editor/Song_Setup#difficulty) and other parameters that directly affect gameplay. Difficulties of a beatmap have different structure and skill requirements and sometimes may be played only in one [game mode](/wiki/Game_mode).
 
-Visual components of a beatmap, contrary to its patterns, provide auxiliary experience during gameplay and can be disabled in the [visual settings](/wiki/Visual_Settings) overlay for the most part. Those include:
+## Beatmap submission
```

Comment:
Consider changing to just "Submission". This goes for all other "Beatmap [word]" headers as well
```suggestion
## Submission
```